govc/test/cluster.bats

```diff
@@ -99,6 +99,65 @@ load test_helper
   run govc cluster.rule.remove -cluster DC0_C0 -name pod1
   assert_success
 
-  run govc cluster.rule.remove -cluster DC0_C0 -name pod1
+  run govc cluster.rule.remove -cluster DC0_C0 -name pod1 -depend
   assert_failure # rule does not exist
+
+  run govc cluster.rule.create -cluster DC0_C0 -name my_deps -depend
+  assert_failure # requires 2 groups
+
+  run govc cluster.rule.create -cluster DC0_C0 -name my_deps -depend my_app my_db
```

Comment:
this is `-depend` or `-depends`?, as well as the 3 cases above this.

Reply:
Good catch, it is `-depends`, those tests were expected to fail, but they were failing because of an invalid flag name.